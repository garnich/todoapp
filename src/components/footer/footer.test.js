import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Footer from './footer'

Enzyme.configure({ adapter: new Adapter() })

describe('Test Footer component', () => {
  it('<Footer>', () => {
    const wrapper = shallow(<Footer />)

    expect(wrapper).toMatchSnapshot()
  })
})
