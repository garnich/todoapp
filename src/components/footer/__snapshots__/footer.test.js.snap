// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer component <Footer> 1`] = `ShallowWrapper {}`;
